zschema/leaves.py

```diff
@@ -246,16 +243,24 @@ class Enum(Leaf):
     INVALID = 23
     VALID = None
 
-    def __init__(self, values=[None,], *args, **kwargs):
+    def __init__(self, values=None, *args, **kwargs):
         Leaf.__init__(self, *args, **kwargs)
+        if values is None:
+            values = []
         self.values = values
         self.values_s = set(values)
 
     def _validate(self, name, value):
-        if value not in self.values_s:
+        if len(self.values_s) and value not in self.values_s:
```

Comment:
I'm not sure if I understand the `len(self.values_s)` clause. Can you elaborate on that?